data-raw/simulating-DAG.R

```diff
@@ -0,0 +1,45 @@
+# A DAG
+library(dagitty)
+library(glasso)
+library(magrittr)
+
+graph <-
+    dagitty("dag{
+        a -> b[beta=.3]
+        b -> c[beta=.2]
+        c -> d[beta=.5]
+        d -> e[beta=.3]
+        e -> f[beta=.6]
+        b [beta=.2]
+        b -> k [beta=.3]
+        k -> l [beta=.3]
+        l -> m [beta=.5]
+        m -> o [beta=.3]
+        o -> p [beta=.5]
+        a -> v [beta=.2]
+        c -> v [beta=.5]
+        d -> v [beta=.3]
+        e -> m [beta=.5]
+        k -> o [beta=.3]
+        o -> p [beta=.65]
+        }")
```

Comment:
Do you want to have an exposure and outcome here? Or are they already included? If so, which variable is the exposure and which is the outcome? To clarify, I don't mean "exposures" and "outcomes" in the context of dagitty, which treats these types of variables differently. I mean, just as a variable name.